Sources/FluentKit/Properties/Siblings.swift

```diff
@@ -34,8 +34,45 @@ public final class Siblings<From, To, Through>: AnyProperty
         return self
     }
 
+    // MARK: Checking state
+
+    public func isAttached(to: To, on database: Database) -> EventLoopFuture<Bool> {
+        guard let toID = to.id else {
+            fatalError("Cannot attach unsaved model.")
+        }
+
+        return isAttached(toID: toID, on: database)
+    }
+
+    public func isAttached(toID: To.IDValue, on database: Database) -> EventLoopFuture<Bool> {
+        guard let fromID = self.idValue else {
+            fatalError("Cannot check if attach siblings relation to unsaved model.")
```

Comment:
This reads a little better:
"Cannot check if siblings are attached to an unsaved model."

Reply:
Ops, I copied and only changes part of the message.